### PR TITLE
Enable CORS headers

### PR DESCRIPTION
**Pull request by sandervankasteel**
As discussed with @morpheus65535 over Discord. Currently this is preliminary PR and RFC for enabling / disabling CORS headers.

This allows browsers to execute cross-domain traffic between their 'domain' (ie. domain1.tld) and the 'domain' Bazarr is running on (192.168.1.1)

When the setting is enabled it will set the HTTP header `Access-Control-Allow-Origin` to `*` (IE, everyone). By default the setting this is disabled. 

More background information on CORS headers: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS

### CODE REVIEW

**Comment by sandervankasteel** [comment]
This PR is currently in draft because I need to finish up on the testing.

**Comment by morpheus65535** [comment]
Looks good but don't add flask-cors to requireents.txt unless it require compiling on client computer. If it's pure python, vendor it in libs directory and make sure to add it to version.txt.

**Comment by sandervankasteel** [comment]
> Looks good but don't add flask-cors to requireents.txt unless it require compiling on client computer. If it's pure python, vendor it in libs directory and make sure to add it to version.txt.

Thanks for your preliminary. I will change this and then continue onward with the testing :smile:

File: bazarr/app/app.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # coding=utf-8
 
+from flask import Flask, redirect
 import os
 
 from flask import Flask
```
Comment (morpheus65535) [inline bazarr/app/app.py:6]: Duplicate import?

**Comment by morpheus65535** [comment]
> @morpheus65535 I've started to continue working on this PR. But I had one question, how do I vendor the `flash-cors` dependency?

Sorry for the delay, I was on vacation.

You can simply put that directory (https://github.com/corydolphin/flask-cors/tree/master/flask_cors) inside `libs` directory.

**Comment by halali** [comment]
This will enable cors only at restart, simple enable setting will do nothing. I will add some note about that fact

File: bazarr/app/app.py (hunk 1 above)
Comment (sandervankasteel) [reply bazarr/app/app.py:6]: Yeah, probably made an error during resolving merge conflicts. Will fix :)

**Comment by sandervankasteel** [comment]
> > @morpheus65535 I've started to continue working on this PR. But I had one question, how do I vendor the `flash-cors` dependency?
> 
> Sorry for the delay, I was on vacation.
> 
> You can simply put that directory (https://github.com/corydolphin/flask-cors/tree/master/flask_cors) inside `libs` directory.

@morpheus65535 I hope you enjoyed your holiday! Thanks for your reply, I will do that. 



> This will enable cors only at restart, simple enable setting will do nothing. I will add some note about that fact

@halali I also made a mention of that in the UI. Source: https://github.com/morpheus65535/bazarr/pull/1868/files#diff-2e987b2ddc76e1053aaf36788a3f14d4dbf315110727e2daf358de9a45a06315R125

**Comment by halali** [comment]
> > > @morpheus65535 I've started to continue working on this PR. But I had one question, how do I vendor the `flash-cors` dependency?
> > 
> > Sorry for the delay, I was on vacation.
> > 
> > You can simply put that directory (https://github.com/corydolphin/flask-cors/tree/master/flask_cors) inside `libs` directory.
> 
> @morpheus65535 I hope you enjoyed your holiday! Thanks for your reply, I will do that. 
> 
> 
> 
> > This will enable cors only at restart, simple enable setting will do nothing. I will add some note about that fact
> 
> @halali I also made a mention of that in the UI. Source: https://github.com/morpheus65535/bazarr/pull/1868/files#diff-2e987b2ddc76e1053aaf36788a3f14d4dbf315110727e2daf358de9a45a06315R125
> 

Sorry, missed that

**Comment by sandervankasteel** [comment]
@morpheus65535 it's ready for review. Sorry for the delay, real life got in the way :)

File: libs/version.txt [hunk 2]
```diff
@@ -8,6 +8,7 @@ dogpile.cache==1.1.5
 enzyme==0.4.1
 fese==0.1.2
 ffsubsync==0.4.20
+flask-cors >= 3.0.10
```
Comment (morpheus65535) [inline libs/version.txt:11]: Should be the actual version:
flask-cors==3.0.10

**Comment by morpheus65535** [comment]
I added one single comment. Other than that, it looks really good, thanks!

File: libs/version.txt (hunk 2 above)
Comment (sandervankasteel) [reply libs/version.txt]: Sharp! I've fixed it :)

**Comment by sandervankasteel** [comment]
@morpheus65535 No problem! And thank you for the review :)

**Pull request merged**